drizzle-orm/src/sqlite-core/session.ts

```diff
@@ -19,6 +25,10 @@ export abstract class PreparedQuery<T extends PreparedQueryConfig> {
 
 	abstract run(placeholderValues?: Record<string, unknown>): Result<T['type'], T['run']>;
 
+	runInBatch(_batch: Batch<unknown, unknown>, _placeholderValues?: Record<string, unknown>): Result<T['type'], T['runBatch']> {
```

Comment:
This should really return `Promise<T['runBatch']>` as it would always be a promise, but when I change it to that lots of type errors appear 😱